AUTHORS

```diff
@@ -976,14 +982,107 @@ Igor Kalashnikov <igor.kalashnikov@me.com>
 Anna Henningsen <anna@addaleax.net>
 Amery <amery@xiangfa.org>
 James Reggio <james.reggio@gmail.com>
+Imran Iqbal <imran@imraniqbal.org>
```

Comment:
Given that these are ordered by date of contribution, I'll just move this entry up and replace the one at line 800